Automattic/pocket-casts-ios
# Settings Sync: App badge

| 📘 Part of: #1400 |
|:---:|

Adds syncing for the App Badge settings.

## To test

Enable the `newSettingsStore` and `settingsSync` feature flags. Restart the app.

* D1: Navigate to Settings > Notifications > App Badge
* D1: Change to different types of App Badge
* D1: Pull to refresh Profile
* D2: Pull to refresh Profile
* D2: Navigate to Settings > Notifications > App Badge
* D2: Verify settings match

You can repeat these steps and check that the App Badge itself is changed after refreshing the Profile as well.

## Checklist
